site/quickstart.md

```diff
@@ -145,15 +147,15 @@ tar
 unzip
 ```
 
-For each WebLogic patches, you will need to download it from Oracle Support and set up the cache. For example, if you downloaded patch number 
+For each WebLogic patch, you will need to download it from Oracle Support and set up the cache. For example, if you downloaded patch number 
 27342434 for WebLogic version 12.2.1.3.0:
  
 ```bash
 imagetool cache addPatch --patchId 27342434 --version 12.2.1.3.0 --path /home/acmeuser/cache/p27342434_122130_Generic .zip -user username@mycompany.com --passwordEnv MYPWD
 ```
 
-You need to provide the credential in the command line, it verifies the MD5 on the file system against the meta data on 
-Oracle Support for this patch number
+You need to provide the credentials in the command line, it verifies the MD5 on the file system against the meta data
```

Comment:
command line, it verifies _> command line; it verifies (you can either separate these two complete sentences with a semi-colon or make them two separate sentences, which probably more correct)